### PR TITLE
Temporaritly remove /gov page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Remove /gov pages
- Add redirect to /desktop/organisations
- Removed from the enterprise mega menu (I updated the copy doc as well)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gov
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to http://0.0.0.0:8001/desktop/organisations

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -457,6 +457,7 @@ gateways/?:  "/internet-of-things/gateways"
 get/?: "/download"
 getubuntu.*/?: "/download"
 GetUbuntu.*/?: "/download"
+gov.*?: "/desktop/organisations"
```
Comment (anthonydillon) [inline redirects.yaml:460]: I think you mean just `gov/?`

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: with a comment

File: redirects.yaml (hunk 1 above)
Comment (pmahnke) [reply redirects.yaml:460]: well, there was a contact-us and thank-you, so I wanted to get that too... do you see any problem with this? too greedy?  I did test on /community/goverance and it seemed ok.

Comment (anthonydillon) [reply redirects.yaml:460]: Ah of course, I wondered if this would pick up `/some/govern/docs` but I guess we don't have gov in other links.

**Pull request merged**